saleor/graphql/core/validators.py

```diff
@@ -9,16 +9,22 @@
     from decimal import Decimal
 
 
-def validate_one_of_args_is_in_query(arg1_name, arg1, arg2_name, arg2):
-    if arg1 and arg2:
+def validate_one_of_args_is_in_query(*args):
+    # split args into 2-element tuples
```

Comment:
For me, it was hard to understand what do you want to achieve. IMO more detailed description will help a lot. Like:
```suggestion
    # split args into a list with 2-element tuples: [(arg1_name, arg1_value), (arg2_name, arg2_value), ...]
```